### PR TITLE
add messagebird to channel capabilities

### PR DESCRIPTION
**Pull request by jpjoyal**
image

### CODE REVIEW

File: data/channelCapabilities.js [hunk 1]
```diff
@@ -331,6 +331,65 @@ const channelData = {
...
+                receive: SUPPORT_LEVEL.NA
+            },
+            readIndicator: {
+                send: SUPPORT_LEVEL.NA,
+                receive: SUPPORT_LEVEL.NA
+            },
+            conversationStart: {
+                receive: SUPPORT_LEVEL.NA
+            },
+            webMessengerLinking: SUPPORT_LEVEL.FULL,
```
Comment (lemieux) [inline data/channelCapabilities.js]: that works? I really don't remember lol

Comment (dannytranlx) [inline data/channelCapabilities.js]: There's an open PR in our main repo. It's being added to the Linking API and the web messenger uses that endpoint IIRC.

Comment (lemieux) [inline data/channelCapabilities.js]: No i meant to not have an object here and just SUPPORT_LEVEL.FULL

Comment (dannytranlx) [inline data/channelCapabilities.js]: Oh, does it work? I don't really remember lol

Comment (jpjoyal) [inline data/channelCapabilities.js]: Yep it works!

Comment (lemieux) [inline data/channelCapabilities.js]: I guess we did it right the first time around :p

**Pull request merged**